src/index.js

```diff
@@ -0,0 +1,50 @@
+let recognizer;
```

Comment:
Dieses File sollte nicht existieren. Wie schon oben gesagt, sollte diese Funktionalität in eine Angular-Komponente.